src/coreComponents/linearAlgebra/unitTests/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ if ( ENABLE_CUDA )
 endif()
 
 if ( ENABLE_PYGEOSX )
-  set( dependencyList ${dependencyList} pygroup pyfileIO pysolver )
+	set( dependencyList ${dependencyList} pygroup pyfileIO pysolver Python3::Python Python3::NumPy pylvarray)
```

Comment:
I'm sorry to see that the situation is so tricky for this merge.

So I have a question, that - who knows - may help you.
I always considered that `pygeosx` should be a wrapper around `geosx`.
_Why would a C++ linear algebra unit test depend on `python`?_
I'm not speaking about _technical considerations_, but from a _macroscopic point of view_.

(Same question applies for all tests here, but I guess it's more striking for linear algebra.)